src/neo4j_graphrag/experimental/pipeline/component.py

```diff
@@ -73,6 +73,7 @@ class Component(abc.ABC, metaclass=ComponentMeta):
 
     # these variables are filled by the metaclass
     # added here for the type checker
+    # DO NOT CHANGE
```

Comment:
What does this refer to?

Reply:
The two lines below the comment

Reply:
I see, what would happen if these were changed?

Reply:
Worst case scenario it would break the pipeline validation. Otherwise they would just be ignored and replaced when the Component class is build, I'm not sure about the resolution order.